src/index.js

```diff
@@ -36,10 +54,12 @@ export default function loader(content, sourceMap) {
 
     const result = node.toStringWithSourceMap({ file: this.resourcePath });
 
+    // @ts-ignore map only has toString() method in types
     callback(null, result.code, result.map.toJSON());
 
     return;
   }
 
+  // @ts-ignore sourceMap in the webpack doesn't have the type RawSourceMap
```

Comment:
We should avoid ignore

Reply:
Seems like the  `callback(null, `${content}\n${FOOTER}${exportsCode}`, sourceMap);` takes type def `SourceMap` that is not exported from webpack. Should we export that and consume it here?

Reply:
This is in webpack 
```ts
declare interface SourceMap {
	version: number;
	sources: string[];
	mappings: string;
	file?: string;
	sourceRoot?: string;
	sourcesContent?: string[];
	names?: string[];
}
```




This is in source-map
```ts
export interface RawSourceMap extends StartOfSourceMap {
    version: string;
    sources: string[];
    names: string[];
    sourcesContent?: string[];
    mappings: string;
}
```

Reply:
which version are you mentioning? webpack? It is in its latest version. I hope the `source-map` is in latest version as well?

Reply:
I think webpack have own types for source maps due to this we have problems with types

Reply:
Bump